plugins/vppv2/puntplugin/README.md

```diff
@@ -1,29 +1,177 @@
 # Punt plugin
 
-Punt plugin defines [model](../model/punt/punt.proto) and supports three scenarios:
+**Written for: v1.7-vpp18.10**
 
-1. **IP punt redirect:** packet matching one of the VPP IP addresses but otherwise would be dropped will be send to 
-the defined next hop IP address via TX interface. The rule can be enhanced and input traffic filtered with RX 
-interface - redirect applies only to traffic received via RX.
+The punt plugin provides several options for how to configure the VPP to allow a specific IP traffic to be punted to the host TCP/IP stack. The plugin supports punt to the host (either directly, or via Unix domain socket) and registration of IP punt redirect rules.
 
-2. **Punt to host:** packet matching one of the VPP IP address and also defined L3 protocol, L4 protocol and also port
-is punted to host
+- [Punt to host stack](#Punt to host stack)
+  * [Model](#pths-model)
+  * [Configuration](#pths-config)
+  * [Limitations](#pths-limit)
+  * [Known issues](#pths-issues)
+- [IP redirect](#IP redirect)
+  * [Model](#ipr-model)
+  * [Configuration](#ipr-config)
+  * [Limitations](#ipr-limit)
+  * [Known issues](#ipr-issues)
 
-3. **Punt to host via socket** packet has to match all the criteria as in previous case, but packet is punted to 
-unix domain socket
+## Punt to host stack
 
-In order to use punt to socket, VPP has to be started with configuration:
+All the incoming traffic matching one of the VPP interface addresses, and also matching defined L3 protocol, L4 protocol, and port - and would be otherwise dropped - will be instead punted to the host. If a Unix domain socket path is defined (optional), the traffic will be punted via socket. All the fields which serve as a traffic filter are mandatory.
+
+### Model
+<a name="pths-model"></a>
```

Comment:
If you want to add some header with custom anchor link, you should define it on the same line as:

```
### <a name="pths-model">Model</a>
```

Reply:
I tried it at first, but this does not render

Reply:
ok, so depending on the parser, somewhere it renders, somewhere not.  Since it works on jbt.github, I'll change it as you proposed